SayTheirNames/Resources/Base.lproj/Localizable.strings

```diff
@@ -34,6 +34,8 @@
 "person.news" = "News";
 "person.media" = "Media";
 "person.hashtags" = "Social Media Hashtags";
+"person.hashtagCopyTip" = "Press and hold to copy";
```

Comment:
This wasn't copied over to the other languages or re-generated by SwiftGen to create the corresponding enum value in L10n. Is that ok ?